src/app/components/find-journey/find-journey.component.html

```diff
@@ -1,6 +1,20 @@
 <div style="text-align:center">
   <h1>
+    <!--lets try to fetch the title from a son-->
     Welcome to {{ title }}!
   </h1>
 </div>
+<form class="example-form" (ngSubmit)="onSubmit()">
+  <mat-form-field class="example-full-width">
+    <input matInput placeholder="Favorite food" value="Sushi">
```

Comment:
matinput?

Reply:
it was in the example. 
I didn't check the doc yet.